liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1832,6 +1844,8 @@ protected void doMigration() throws Exception {
 
             try {
                 if (COMMANDS.UPDATE.equalsIgnoreCase(command)) {
+                    Scope.getCurrentScope().addMdcValue(MdcKey.LIQUIBASE_COMMAND_NAME, COMMANDS.UPDATE);
```

Comment:
Can you put the line for LIQUIBASE_COMMAND_NAME a level higher, and use the `command` variable instead of `COMMANDS.UPDATE`? Then we'd get this for free for most commands.

Reply:
Great idea, updated!